app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, start, end = command.split()
```

Comment:
Validate command length before splitting

Validate that the command name is exactly "cp"